extensions/map-layers/package.json

```diff
@@ -44,7 +44,7 @@
     "@bentley/imodeljs-quantity": "2.14.0-dev.16",
     "@bentley/itwin-client": "2.14.0-dev.16",
     "@bentley/presentation-common": "2.14.0-dev.16",
-    "@bentley/react-scripts": "3.4.9",
+    "@bentley/react-scripts": "4.0.3-dev.0",
```

Comment:
Does this version of react-scripts include the fix in this PR for the updated version of immer: https://github.com/facebook/create-react-app/issues/10411

Reply:
it does :)